schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE SCHEMA `vet_clinic` ;
+CREATE TABLE `vet_clinic`.`animals` (
+  `idAnimals` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `date_of_birth` DATETIME NOT NULL,
+  `escape_attempts` INT NOT NULL,
+  `neutered` TINYINT(1) NOT NULL,
```

Comment:
- [ ] The `TINYINT` keyword gives an error when I tried to create the table. Also, the `neutered` column should be a `BOOLEAN`. Kindly make this change. 👍